specifications/dpe-irot-profile/spec.ocp

```diff
@@ -1468,9 +1621,24 @@ Table: `SIGN_INPUT_ARGS` struct
 | 0x08            | `U32`         | 31:0     | `PROFILE`            | One of `DPE_PROFILE_IROT_SHA256_P256`.
 | 0x0C            | `BYTES`       | 127:0    | `CONTEXT_HANDLE`     | A numeric handle referring to a DPE context.
 | 0x1C            | `HASH`        |          | `LABEL`              | Digest measurement label used in key derivation.
-| 0x1C + H        | `BITFIELD`    | 31:0     | `RESERVED`           | Reserved
+| 0x1C + H        | `BITFIELD`    | 31       | `FULL_MESSAGE`       | If set, `TO_BE_SIGNED` contains the full message. This flag is only supported where explicitly specified in `to-be-signed-format`.
```

Comment:
I'm not yet convinced this is the best way to do this, but I kinda like it because it doesn't break back compat with older profile revisions.